Emby.Server.Implementations/Activity/ActivityManager.cs

```diff
@@ -49,10 +69,5 @@ public QueryResult<ActivityLogEntry> GetActivityLogEntries(DateTime? minDate, bo
 
             return result;
```

Comment:
I would suggest to materialize the result with e.g. `result.ToList()` if `IEnumerable` is being returned or return an `IQueryable` instead (which is probably not what you want in this layer)

Reply:
Nope, the count will generate a SQL query similar to SELECT COUNT (*) FROM ActivityLogEntries, whereas the one for the loop will result in something like SELECT * FROM ActivityLogEntries.
EF has no way to know it should execute the actual query because it will be re-used further down the execution path

Reply:
Oh right, cause it's still an `IQueryable` under the hood.